rules/go_rules.build_defs

```diff
@@ -108,8 +112,9 @@ def go_library(name:str, srcs:list, asm_srcs:list=None, hdrs:list=None, out:str=
             building_description = 'Packing...',
             requires = ['go'],
             labels = labels,
-            provides = provides,
+            provides =  {'go': ':' + name, 'go_src': lib_rule},
             test_only = test_only,
+            exported_deps = exported_deps,
```

Comment:
```suggestion
            exported_deps = exported_deps + deps,
```

Not really nice, but fixes #1181

Reply:
Nice spot! They don't need to be exported, they just need to be deps which is nice.